libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/JSON.hs

```diff
@@ -58,3 +59,13 @@ roundTripJsonEraSpec =
   describe (eraName @era) $ do
     describe "RoundTrip JSON" $ do
       roundTripJsonSpec @(PParams era)
+
+goldenPParamsSpec ::
```

Comment:
I think it will be nice if add a suffix, then it is a bit more clear what type of golden testing we are doing at the call site
```suggestion
goldenJsonPParamsSpec ::
```